### PR TITLE
Add primary wallet feature to web

### PR DESCRIPTION
**Pull request by Lerek**

### CODE REVIEW

**Review by Riddlerrr — CHANGES_REQUESTED** [review]
Let's also use this new `MakePrimaryWallet` class into `Api::V1::WalletsController#update`:
https://github.com/CoMakery/comakery-server/blob/acceptance/app/controllers/api/v1/wallets_controller.rb#L27

File: app/controllers/primaries_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class PrimariesController < ApplicationController
```
Comment (Riddlerrr) [inline app/controllers/primaries_controller.rb:1]: 🤔 I think in this case better to simply add `set_primary` action to `WalletController`
It should improve readability. I can't guess what the controller does just by its name.

Comment (Lerek) [reply app/controllers/primaries_controller.rb:1]: @Riddlerrr maybe then it's better just to rename the controller to `WalletPrimariesController` to keep REST style, what do you think about it?

Comment (Riddlerrr) [reply app/controllers/primaries_controller.rb:1]: I thought about it too but my vote still to `set_primary` action. I don't think REST is so important in this case, it is not an API. And this controller will be always with just one action for sure.

@Alexey1100 What do you think?

Comment (Lerek) [reply app/controllers/primaries_controller.rb:1]: @Riddlerrr nvm, I do not insist :) I will move the logic to new action

**Review by Riddlerrr — APPROVED** [review]
👍  Looks good to me

**Comment by Riddlerrr** [comment]
@aquabu It is ready to merge

**Review by aquabu — APPROVED** [review]

**Pull request merged**